packages/terra-clinical-header/src/Header.scss

```diff
@@ -39,6 +39,14 @@
     position: relative;
   }
 
+  .start-content {
+    margin-right: 5px;
```

Comment:
These should be updated to be rem values

Reply:
This needs design input; I'll update it as soon as I have a rem value from UX.